Headway/BookPlayerView.swift

```diff
@@ -0,0 +1,153 @@
+import ComposableArchitecture
+import SwiftUI
+
+struct BookPlayerView: View {
+    @Bindable var store: StoreOf<BookPlayerFeature>
+
+    @State private var localSliderValue: Double? = nil
+
+    var body: some View {
+        NavigationView {
+            if store.isLoading {
+                ProgressView("Loading Book…")
+            } else if let book = store.book {
+                VStack(spacing: 48) {
+                    playerImage(for: book)
+
+                    keyPointView(for: book)
+                        .padding(.horizontal)
+
+                    progressView
+
+                    controlButtons
+
+                    Spacer()
+                }
+                .padding(.horizontal)
+                .background(Color(.secondarySystemBackground))
+            }
+        }
+        .alert($store.scope(state: \.alert, action: \.alert))
+        .onAppear { store.send(.loadBook) }
+    }
+
+    private func playerImage(for book: Book) -> some View {
+        Image(resource: book.coverFileName)?
+            .resizable()
+            .scaledToFit()
+            .frame(width: 220)
+            .cornerRadius(10)
+    }
+
+    private func keyPointView(for book: Book) -> some View {
+        VStack(spacing: 8) {
+            Text("KEY POINT \(store.selectedChapterIndex + 1) OF \(book.chapters.count)")
+                .font(.footnote)
+                .foregroundColor(.gray)
+
+            Text(book.chapters[store.selectedChapterIndex].keyPoint)
+                .font(.callout)
+                .multilineTextAlignment(.center)
+        }
+        .foregroundStyle(Color.primary)
+    }
+
+    private var progressView: some View {
+        VStack(spacing: 16) {
+            HStack(alignment: .center) {
+                Text(formatTime(store.playbackProgress * store.duration))
+                    .font(.caption)
+                    .foregroundColor(.secondary)
+                    .frame(width: 44)
+
+                Slider(
+                    value: Binding(
+                        get: { localSliderValue ?? store.playbackProgress },
+                        set: { localSliderValue = $0 }
+                    ),
+                    in: 0 ... 1,
+                    onEditingChanged: { editing in
+                        if !editing, let finalValue = localSliderValue {
+                            store.send(.seek(finalValue))
+                            localSliderValue = nil
+                        }
+                    }
+                )
```

Comment:
More straighforward approach would be to use basic binding:
```
Slider(
    value: $store.playbackProgress.sending(\.seek),
    in: 0 ... 1
)
```
but it produces a very inconsistent behaviour with a jumbping slider. Two-way binding are always difficult. This approach is less vanilla, but slider is much smoother.